_includes/scout-plans.md

```diff
@@ -0,0 +1,3 @@
+> **Info**
+>
+> The Docker Scout free plan gives you unlimited local image analysis and up to 3 remote repositories. [If you need more, upgrade your plan](#).
```

Comment:
The PR with the Scout billing and subscription pages is still in review so this page isn't merged to main yet, but here is the path:
```suggestion
> The Docker Scout free plan gives you unlimited local image analysis and up to 3 remote repositories. [If you need more, upgrade your plan](/billing/scout-billing.md).
```